perlin.py

```diff
@@ -46,12 +46,12 @@ def get_perlin_noise(x: float, y: float, base: int, scale: float = 0.005, octave
     return (noise.pnoise3(x=x * scale, y=y * scale, z=base, octaves=octaves) + 1) / 2
 
 
-def get_population_number(edge: sumolib.net.edge.Edge, base: int, centre,
-                          radius, centre_weight: float = 1.0, scale: float = 0.005, octaves: int = 3) -> float:
+def get_noise_interval(edge: sumolib.net.edge.Edge, base: int, centre,
```

Comment:
It doesn't return an interval. Alternatives:
`get_noise_sample`
`sample_edge_noise`

Reply:
Fixed in d4c691d